File: Python Codes/PrimeNumber.py
import math
num = int(input("Enter a number: "))
if num==1:
    print("Non prime")
    exit()
i = 2
isPrime = True
while i<=(int(math.sqrt(num))):
    print(i)
    if num%i==0:
        isPrime = False
        break
    i += 1
if isPrime:
    print("Prime")
else:
    print("Non prime")

File: C program/New Batch 2/PrimeNumber.cpp
#include<stdio.h>
#include<math.h>
int main(){
    int num;//10
    printf("Enter a number: ");
    scanf("%d",&num);
    int i;
    for(i=2;i<=sqrt(num);i++){
        if(num%i==0){
            break;
        }
    }
    if(i>sqrt(num)){
        printf("Prime number");
    }
    else{
        printf("Non prime number");
    }
    return 0;
}